Library/Homebrew/cli/named_args.rb

```diff
@@ -50,13 +50,15 @@ def to_formulae
           method:                  T.nilable(Symbol),
           uniq:                    T::Boolean,
           prefer_loading_from_api: T::Boolean,
+          only_unmigrated:         T::Boolean,
```

Comment:
It might be that this isn't needed (though I have this feeling this might be useful elsewhere), but I wonder if we should instead be able to pass (semi-)arbitrary predicates here (perhaps expressed as `Symbol`s) to filter formulae and casks rather than adding `Boolean` kwargs for each one.